namespace/namespace.go

```diff
@@ -33,7 +33,7 @@ func NewNamespaceCommands() []*cli.Command {
 			},
 		},
 		{
-			Name:      "register",
+			Name:      "create",
 			Usage:     "Register a new Namespace",
 			Flags:     registerNamespaceFlags,
```

Comment:
Flags variable should be renamed to `createNamespaceFlags`

Reply:
updated